### PR TITLE
Tg feature onboardingasync

### PR DESCRIPTION
**Pull request by talgrewal**
# Create onboarding async

- Add touchable opacity to go from onboarding screen to login page
- Add async storage to track if onboarding has happened
- Add logic to not show onboarding if value is true

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (siddharthparmar7) [inline yarn.lock]: delete this file.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
delete .DS_Store file and yarn lock from the root of your project.

**Comment by siddharthparmar7** [comment]
Also, please resolve merge conflicts.

File: yarn.lock (hunk 1 above)
Comment (talgrewal) [reply yarn.lock]: Done.

**Comment by talgrewal** [comment]
The push for this pull request failed. Will need assistance in pushing on Monday.

File: client/js/components/MainSubscribeButton/MainSubscribeButton.js [hunk 2]
```diff
@@ -36,9 +39,13 @@ const MainSubscribeButton = ({CampaignId, ViewerId, userId}) => {
                     variables: {
                       campaignid: CampaignId,
 
+
                       userid: userId,
+
                     },
                   });
+                  console.log('hello');
```
Comment (siddharthparmar7) [inline client/js/components/MainSubscribeButton/MainSubscribeButton.js]: remove this console log.

File: client/js/screens/Onboarding/Onboarding.js [hunk 3]
```diff
@@ -64,13 +79,36 @@ export default class Onboarding extends Component {
...
         renderDoneButton={() => {
-          return <Button style={styles.done} title="Join the Revolution" />;
+          return (
+            <TouchableOpacity
+              onPress={async () => {
+                try {
+                  await onBoardingSet(true);
+                  this.props.navigation.navigate('Login');
+                } catch (e) {
+                  console.log(e);
```
Comment (siddharthparmar7) [inline client/js/screens/Onboarding/Onboarding.js:100]: remove this console log.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts.

File: client/js/screens/Onboarding/Onboarding.js (hunk 3 above)
Comment (talgrewal) [reply client/js/screens/Onboarding/Onboarding.js:100]: Done.

File: client/js/components/MainSubscribeButton/MainSubscribeButton.js (hunk 2 above)
Comment (talgrewal) [reply client/js/components/MainSubscribeButton/MainSubscribeButton.js]: Done

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please remove .DS_Store file from the root of your project.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**